include/sys/arc.h

```diff
@@ -18,10 +18,13 @@
  *
```

Comment:
same

Reply:
This one has a reasonable amount of copyrighted code from allan afaik.